ui/core/individual_sim_ui.tsx

```diff
@@ -363,6 +364,17 @@ export abstract class IndividualSimUI<SpecType extends Spec> extends SimUI {
 		});
 	}
 
+	// Did this to get rid of the TypeError of the Manager getting passed as null, but this abomination would probably break more things than fix :P
+	// private initializeRaidSimResultsManager() {
+	// 	try {
+	// 		this.raidSimResultsManager = new RaidSimResultsManager(this);
+	// 		console.log('raidSimResultsManager initialized:', this.raidSimResultsManager);
+	// 	} catch (error) {
+	// 		console.error('Failed to initialize raidSimResultsManager:', error);
+	// 		this.raidSimResultsManager = null;
+	// 	}
+	// }
```

Comment:
Hey @jjayrex would you mind cleaning up these comments?

Reply:
@kayla-glick Removed all the changes there as they didn't really impact anything here, I'll keep a local copy in case I try to do something with it